.github/workflows/release.yml

```diff
@@ -46,3 +46,9 @@ jobs:
           prerelease: ${{ steps.changelog_reader.outputs.status == 'prereleased' }}
           draft: ${{ steps.changelog_reader.outputs.status == 'unreleased' }}
           token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Push Lambda Extension Layer Image
+        env:
+          CRYPTEIA_VERSION: ${{ steps.changelog_reader.outputs.version }}
+        run: |
+          echo "${{ secrets.PUBLIC_GITHUB_TOKEN_PACKAGES }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+          ./layer/deploy-image
```

Comment:
Using the new `PUBLIC_GITHUB_TOKEN_PACKAGES` secret (cc @shellerci) for our open source repos. This will push the Amazon Linux 2 container image along with the other release binary and shared object files. Hmm... typing this out, I think I should add `-amzn` suffix here too and make a `debian` one.